win32ss/gdi/ntgdi/text.h

```diff
@@ -144,3 +144,10 @@ BOOL WINAPI GreGetTextMetricsW( _In_  HDC hdc, _Out_ LPTEXTMETRICW lptm);
 
 #define IntUnLockProcessPrivateFonts(W32Process) \
   ExReleaseFastMutexUnsafeAndLeaveCriticalRegion(&W32Process->PrivateFontListLock)
+
+VOID DumpFontGDI(PFONTGDI FontGDI);
+VOID DumpGlobalFontList(BOOL bDoLock);
+VOID DumpPrivateFontList(BOOL bDoLock);
+//VOID DumpFontSubstEntry(PFONTSUBST_ENTRY pSubstEntry);
+VOID DumpFontSubstList(VOID);
+VOID DumpFontInfo(BOOL bDoLock);
```

Comment:
As long as they are only called from freetype.c, they do not need to be here?

Reply:
Reverted.

Reply:
"win32ss/gdi/ntgdi/font.c" uses PFONTGDI.